config/src/main/java/org/springframework/security/config/annotation/web/configurers/PermitAllSupport.java

```diff
@@ -89,11 +90,11 @@ public boolean matches(HttpServletRequest request) {
 		@Override
 		public String toString() {
 			StringBuilder sb = new StringBuilder();
-			sb.append("ExactUrl [processUrl='").append(processUrl).append("']");
+			sb.append("ExactUrl [processUrl='").append(processUrl).append("', id=").append(getId()).append("]");
```

Comment:
I think we can live away with String builder now, by-default string concatenations internally invoke string builder only. Plus it will result in cleaner code